frontend/src/components/ModalContent.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+  <v-card>
+    <v-card-title id="title-header">
+      <v-row>
+        <p id="modal-title">{{ selectedProject.name }}</p>
+        <v-card-actions>
+          <v-btn
+            class="mr-2"
+            absolute
+            fab
+            rounded
+            icon
+            large
+            right
+            @click="onClick()"
+          >
+            <v-icon size="3.5rem" color="white">close</v-icon>
+          </v-btn>
+        </v-card-actions>
+      </v-row>
+    </v-card-title>
+    <v-card-text>
+      <v-card-actions>
+        <v-btn text rounded :href="selectedProject.licensePage" target="_blank">
+          <v-icon>far fa-copyright</v-icon>
+          <p class="buttons">Licencja {{ selectedProject.licenseName }}</p>
+        </v-btn>
+        <v-btn text rounded :href="selectedProject.websiteLink" target="_blank">
+          <v-icon>language</v-icon>
+          <p class="buttons">Strona WWW</p>
+        </v-btn>
+        <v-btn text rounded :href="selectedProject.githubLink" target="_blank">
+          <v-icon>fab fa-github</v-icon>
+          <p class="buttons">Repozytorium</p>
+        </v-btn>
+      </v-card-actions>
+    </v-card-text>
+    <v-card-text>
+      <v-row class="ml-3 mr-3">
+        <p class="modal-subtitle">
+          Partner projektu:
+        </p>
+        <v-btn text rounded :href="selectedProject.partnerPage" target="_blank">
+          <a class="modal-subtitle">{{ selectedProject.partner }}</a>
+        </v-btn>
+      </v-row>
+      <v-row>
+        <p class="content--black">{{ selectedProject.description }}</p>
+      </v-row>
+      <v-row class="ml-3 mr-3">
+        <p class="modal-subtitle">Wykorzystane technologie:</p>
+      </v-row>
+      <v-row class="mr-1 ml-1">
+        <v-btn
+          class="stack-list ma-2"
+          v-for="(item, index) in selectedProject.stack"
+          :key="index"
+          :href="item.documentation"
+          target="_blank"
+          rounded
+        >
+          {{ item.type }}: {{ item.name }} {{ item.version }}
+        </v-btn>
+      </v-row>
+    </v-card-text>
+  </v-card>
+</template>
+
+<script>
+export default {
+  props: ['selectedProject'],
+  methods: {
+    onClick() {
+      this.$root.$emit('close');
+    }
+  }
+};
+</script>
+
+<style lang="scss" scoped>
+@import './../main.scss';
+
+a {
+  text-transform: none;
+}
+
+.buttons {
+  display: flex;
+  align-content: center;
+  margin: 0.5rem;
+}
+
+#close-button {
```

Comment:
Do we want to use ids? They are quite dangerous, because they don't work when there are 2 same ids on the same page